src/TrustedOracle.sol

```diff
@@ -5,14 +5,23 @@ import "openzeppelin-contracts/contracts/access/Ownable.sol";
 import "./interfaces/ITrustedOracle.sol";
```

Comment:
Can we please rename the `ITrustedOracle` interface to `IBlockhashStorage`, to avoid the word oracle in the interface